### DISCUSSION TITLE
SSH - non-discoverable ed25519-sk keys

### DISCUSSION THREAD

**Discussion by BarsMonster** [solokeys/solo2#169] [General]
I managed to make discoverable (resident) keys work (ed25519-sk) with Putty-CAC (Windows 11) and Solo2. 
Unfortunately, non-discoverable keys do not work. I got asked for a PIN, and then - I get error "This security key doesn't look familiar. Please try a different one". Any tricks needed to make non-discoverable (and "no touch") keys to work?

![image](https://github.com/solokeys/solo2/assets/4014665/10750693-787e-4e22-916f-c07b0dd85fbf)

**Comment by DkJan** [comment]
putty and openssh use different file formats to store the same key.
If you used openssh ssh-keygen to generate the key and want putty to use it - you usually have to use the "PuTTY Key Generator" and import the openssh key and save it in the putty format. Short googling from my side gave no relevant results on "putty ed25519-sk" for me. I have no clue whether putty supports this format which is in use since version 8.2 of openssh....

**Comment by BarsMonster** [comment]
@DkJan Only Putty-CAC has a chance to support it, not regular Putty. It has dedicated function to import ed25519-sk keys from file.
